src/main/java/com/gradle/Utils.java

```diff
@@ -34,22 +33,6 @@
         return Optional.ofNullable(System.getProperty(name));
     }
 
-    static Optional<Boolean> booleanSysProperty(String name) {
-        return sysProperty(name).map(Boolean::parseBoolean);
-    }
-
-    static Optional<Duration> durationSysProperty(String name) {
-        return sysProperty(name).map(Duration::parse);
```

Comment:
I would have kept the two methods above (even though currently unused) since they are generic enough that someone might clone the project and use them. But, I do not feel too strongly.